lib/features/guide/widgets/about_us/models/member_data.dart

```diff
@@ -0,0 +1,20 @@
+import 'link_data.dart';
+
+class MemberData {
```

Comment:
so `freezed` is best practice for models, but here it may be impossible/hard to implement (due to a bit weird LinkData class), so that's not a big problem 

but still at least do the following:
1. add `final` to all attributes
2. annotate `immutable` to the class:
```dart
@immutable
class MemberData {
  final List<String> socialLinks;
  final String? name;
  final String? subtitle;
  final List<LinkData> links;
  final String? imageUrl;

//...
```

that's the minimum, for fully proper model you would need to override `==` operator and `hashCode`, but as long it's not fully needed, you can skip it. `freezed` does that for us, but here it may be hard to do, cause we don't want to iterate over the all entries of the map with icons. So just do the 2 things noted above